pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog@id.md

```diff
@@ -78,7 +78,7 @@ Anda dapat menambahkan atribut yang dinonaktifkan ke halaman yang dihosting untu
 
 ### Deeplinking
 
-Saat Anda mempublikasikan entri blog, penting untuk menempatkan deep link ke postingan untuk mengaktifkan fitur seperti berbagi. Dengan [`amp-live-list`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-live-list.md', locale=doc.locale).url.path}}), deep linking dapat dilakukan dengan hanya menggunakan` id` dari item blog. Misalnya, [https://ampbyexample.com/samples_templates/live_blog/preview/#post3](https://ampbyexample.com/samples_templates/live_blog/preview/#post3) mengizinkan Anda membuka postingan blog secara langsung dengan ID `post3`.
+Saat Anda mempublikasikan entri blog, penting untuk menempatkan deep link ke postingan untuk mengaktifkan fitur seperti berbagi. Dengan [`amp-live-list`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-live-list.md', locale=doc.locale).url.path}}), deep linking dapat dilakukan dengan hanya menggunakan` id` dari item blog. Misalnya, [https://amp.dev/documentation/examples/news-publishing/live_blog/preview/index.html#post3]({{g.doc('/content/amp-dev/documentation/examples/previews/Live_Blog.html', locale=doc.locale).url.path}}#post3) mengizinkan Anda membuka postingan blog secara langsung dengan ID `post3`.
 
 AMP By Example menggunakan cookie di [contoh blog langsung]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md', locale=doc.locale).url.path}}) untuk membuat konten terkini. Karena itu, jika ini pertama kalinya Anda membuka halaman, postingan dengan ID “post3” mungkin tidak tersedia, dan Anda akan diarahkan ke postingan pertama.
```

Comment:
Please check for all occurences of "AMP By Example" as well while we are at it!

Reply:
@sebastianbenz, what do you think? I remember in our call we quickly were on the same page rewriting all references to AMP by Example but @robinvanopstal just figured out that might not be too easy as just replacing "AMP by Example" with "Examples" will sometimes be double as in "see examples at ~~AMP by Example~~Examples". Not to mention i18n.

Reply:
Yes, these need to be manually checked and rephrased.

Reply:
... and leave the translations dirty for now?